<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1589</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1581</width>
      <height>871</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalContainer">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsViewResult"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="percentageDropdown">
         <item>
          <property name="text">
           <string>25%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>75%</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeSizeSubmit">
         <property name="text">
          <string>Change Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="imageDropDown">
         <property name="currentText">
          <string notr="true">FaceBook 170x170 </string>
         </property>
         <item>
          <property name="text">
           <string>FaceBook 170x170 </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cover Photo: 820 x 312</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Profile Picture: 110 x 110</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Photo Thumbnail: 161 x 161</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Photo Size: 1080 x 1080</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Instagram Story: 1080 1920</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shared Image: 1200 x 630</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shared link: 1200 x 628</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Event Image: 1920 x 1005</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fundraiser image: 800 x 300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Profile Photo: 400 x 400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Header Photo: 1500 x 500</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="imageSizeSubmit">
         <property name="text">
          <string>Change Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cropButton">
         <property name="text">
          <string>Crop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1589</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/imgs/Open.bmp</normaloff>:/imgs/Open.bmp</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>:/imgs/Save.bmp</normaloff>:/imgs/Save.bmp</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
